### DISCUSSION TITLE
VolumeSnapshot in CephFS

### DISCUSSION THREAD

**Discussion by ahmedTouati** [rook/rook#9285] [Q&A]
Is it a normal behaviour that the volume snapshot in cephFS occupies the same size as the original volume?

**Comment by BlaineEXE** [comment]
I can't find any solid documentation explaining how CephFS snapshots are stored with respect to data size. Obviously, a snapshot doesn't have a non-zero size. Generally speaking, I would expect the first snapshot of any system to be a full-size copy, but subsequent snapshots could be smaller deltas if the snapshots are space efficient.

@Madhu-1 and @leseb might have more experience with CephFS snapshots to provide more definite info.

**Reply by leseb** [reply]
I don't but I convey @BlaineEXE's assumption about the size. Maybe @ajarr knows more? Thanks!

**Reply by Madhu-1** [reply]
AFAIK cephfs snapshots are always full-copy. @kotreshhr can help
